application/src/main/java/com/sanction/thunder/authentication/UserAuthenticator.java

```diff
@@ -0,0 +1,40 @@
+package com.sanction.thunder.authentication;
+
+import com.google.common.base.Optional;
+import com.sanction.thunder.dao.PilotUsersDao;
+import com.sanction.thunder.models.PilotUser;
+import io.dropwizard.auth.AuthenticationException;
+import io.dropwizard.auth.Authenticator;
+import io.dropwizard.auth.basic.BasicCredentials;
+
+import javax.inject.Inject;
+
+public class UserAuthenticator implements Authenticator<BasicCredentials, User> {
```

Comment:
Not sure about this but I want your opinion - do we want to call this `ThunderAuthenticator` instead in order to follow the convention we had before?